tests/integration/components/o-s-s/panel-row-test.ts

```diff
@@ -20,4 +20,9 @@ module('Integration | Component | o-s-s/panel-row', function (hooks) {
     await render(hbs`<OSS::Panel::Row @icon={{"fa-cog"}} @label={{'Your Label'l}} />`);
     assert.dom('span').hasText('Your Label');
   });
+
+  test('it renders as disabled when @disabled is provided', async function (assert) {
+    await render(hbs`<OSS::Panel::Row @icon={{"fa-cog"}} @label={{'Your Label'l}} @disabled={{true}}/>`);
```

Comment:
small typo here, there is a l after the Your Label closing quote

Reply:
Fixed typo , thanks 👍